test

```diff
@@ -83,9 +88,13 @@ function run_pipeline_tests()
     # Check if we have a composer.json so we can run tests
     # Check inside the console container as static builds do not
     # update the local filesystem
-    if ws exec stat composer.json > /dev/null 2>&1; then
+    if ws exec stat /app/composer.json; then
```

Comment:
why is the silencing of output removed?

Reply:
Was for debugging as it wasn't firing the tests every time. Fixed :+1: